stable/metrics-server/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.2.1
 description: Metrics Server is a cluster-wide aggregator of resource usage data.
 name: metrics-server
-version: 0.0.2
+version: 1.0.0
```

Comment:
looks like this should be a minor version bump rather than a major as its backwards compatible and doesnt change any behavior of the default install. 0.1.0?

Reply:
The reason for bumping to 1.0.0 is that the chart is stable, at least with #6488 merged in as well. But I can minor bump on this one and keep the major bump in the referenced one.